rustls/examples/internal/test_ca.rs

```diff
@@ -138,12 +141,25 @@ fn crl_for_serial(serial_number: SerialNumber) -> CertificateRevocationListParam
     }
 }
 
+fn expired_crl() -> CertificateRevocationListParams {
+    let now = OffsetDateTime::now_utc();
+    CertificateRevocationListParams {
+        this_update: now,
+        next_update: now + Duration::from_secs(60),
```

Comment:
Ah, I guess I made rcgen smart enough to reject `next_update` values in the past eh? 

I think this is a reasonable workaround but might be worth a comment explaining it.

Reply:
Actually I didn't even try it. I've updated to use a `this_update` value in the past, and a `next_update` value with current time, which worked fine.